actions_std.go

```diff
@@ -2160,6 +2160,21 @@ func scriptingActions() {
 			}
 		},
 	}
+	actions["openApp"] = actionDefinition{
+		identifier: "openapp",
+		parameters: []parameterDefinition{
+			{
+				name:      "appIdentifier",
+				validType: String,
+				noMax:     true,
+			},
+		},
+		make: func(args []actionArgument) []plistData {
+			return []plistData{
+				argumentValue("WFSelectedApp", args, 0), // WFAppIdentifier Previously - changed in iOS 15(?)
```

Comment:
`WFAppIdentifier` belongs here instead.

`WFSelectedApp` is a dictionary with a `BundleIdentifier` with the same value as `WFAppIdentifier`.

```go
{
	key:      "WFSelectedApp",
	dataType: Dictionary,
	value: []plistData{
		argumentValue("BundleIdentifier", args, 0),
	},
},
```

Please remove the comment.

[Please confirm action parameters with testing before submitting a PR.](https://github.com/electrikmilk/cherri/blob/main/CONTRIBUTING.md)